test/promql/tests/application_errors_test.yaml

```
@@ -0,0 +1,102 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.application_alerts.yaml
+
+tests:
+  # Alerted cases:
+  - interval: 5m
```

This one (same for all test cases) defines the increments of the series that you provide, so it should be set to 1m to correspond with the series you provide and the evaluation time of the rule.
```suggestion
  - interval: 1m
```